apps/mosaico/src/Main.purs

```diff
@@ -679,19 +678,27 @@ renderCategoryPage env (Category { label, type: categoryType, url}) = do
              , latestArticles
              }
 
-  let htmlTemplate = cloneTemplate env.htmlTemplate
-      mosaicoString = renderContent <$> mainContent <*> mostReadArticles <*> latestArticles
-      windowVars = stdVars env mostReadArticles latestArticles
-                   <> foldMap (mkArticleFeed $ CategoryFeed label) feed
-  html <- liftEffect $ appendMosaico (Cache.getContent mosaicoString) htmlTemplate >>=
-          appendVars (mkWindowVariables windowVars) >>=
-          appendHead (makeTitle (unwrap label))
-  let rendered = renderTemplateHtml html
-  Cache.saveCategoryRender env.cache label $ mosaicoString $> rendered
-  now <- liftEffect nowDateTime
-  pure $ Cache.addHeader now mosaicoString $
-    htmlContent $ Response.ok $ StringBody rendered
+  case categoryType of
+    -- Fallback to using list feed style
+    Prerendered
+      | Just (Html html _) <- feed
+      , Right regex <- emptyRegex
+      , Just _ <- Regex.match regex html -> renderCategoryPage env $ Category $ category { type = Feed }
```

Comment:
How do we get an empty string as `html`?

Reply:
It had a `fromMaybe ""` previously, but it wasn't that hard to bring it as a `Nothing` to that case since feed was a `Maybe` already.

I changed it but I still kept the test for empty string since it feels like a good sanity check to have.